usage-collection/src/main/java/com/sequenceiq/cloudbreak/usage/metering/MeteringDatabusRecordProcessor.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.cloudbreak.usage.metering;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.auth.crn.RegionAwareInternalCrnGeneratorFactory;
+import com.sequenceiq.cloudbreak.sigmadbus.config.SigmaDatabusConfig;
+import com.sequenceiq.cloudbreak.sigmadbus.processor.AbstractDatabusRecordProcessor;
+import com.sequenceiq.cloudbreak.telemetry.metering.MeteringConfiguration;
+
+import io.opentracing.Tracer;
+
+@Component
+public class MeteringDatabusRecordProcessor extends AbstractDatabusRecordProcessor<MeteringConfiguration> {
+
+    public MeteringDatabusRecordProcessor(SigmaDatabusConfig sigmaDatabusConfig, MeteringConfiguration meteringConfiguration,
```

Comment:
im wondering it would make sense to override the isDatabusProcessingEnabled method (or extend the configuration object, and override enabled there) to add some additional configs to make sure consumption stuff is also enabled (otherwise we will start a worker thread which won't do anything if metering is enabled...this won't hurt that much, but can be avoided)

Reply:
I created a MeteringConsumptionConfiguration config, extending the MeteringConfiguration, that has a separate consumptionEnabled field and overrides the isEnabled() function to also check if the consumption is enabled. It was discussed in the consumption sync today that it's fine for now to just leave it as true and add a change later to disable it by default and enable in every non-local environment. The follow up ticket for this is CB-17470.